property.go

```diff
@@ -13,29 +14,13 @@ type Property interface {
 	GetType() PropertyType
 }
 
-type PropertyID string
-
-func (pID PropertyID) String() string {
-	return string(pID)
-}
-
-type TextProperty struct {
+type TitleProperty struct {
 	ID    PropertyID   `json:"id,omitempty"`
 	Type  PropertyType `json:"type"`
```

Comment:
Shouldn't `Type` be `omitempty` ? Or those structs should always be marshalled with they `Type` field inferred from the Go type instead? I feel that's without one of these two changes, that's not very practical and can lead the user to very predictable mistakes that are easily fixed.  

```suggestion
	Type  PropertyType `json:"type,omitempty"`
```

I tested it on my side, it works as intended with that change. 

---

Go struct: 

```go
	properties["Name"] = notionapi.TitleProperty{
		Title: []notionapi.RichText{{Text: notionapi.Text{Content: item.Name}}},
	}
```

What's being sent: 
```
{"Name":{"type":"","title":[{"text":{"content":"Foo"}}]}}
``` 
API answer:

```
2021/08/01 12:18:42 cannot insert item: request error: body failed validation. Fix one: body.properties.Name.id should be defined, instead was `undefined`. body.properties.Name.name should be defined, instead was `undefined`. body.propertie
```